.github/workflows/deploy.yml

```
@@ -0,0 +1,29 @@
+        uses: appleboy/ssh-action@v1.0.3
+        with:
+          host: ${{ inputs.host }}
+          username: ${{ secrets.DEPLOY_USER }}
+          key: ${{ secrets.DEPLOY_KEY }}
+          script: |
+            set -e
+            mkdir -p ${{ inputs.path }}
+            rsync -a --delete . ${{ secrets.DEPLOY_USER }}@${{ inputs.host }}:${{ inputs.path }}
+            cd ${{ inputs.path }}
+            docker compose pull || true
+            docker compose up -d --build
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)